packages/rest/src/lib/utils/constants.ts

```diff
@@ -12,13 +11,7 @@ export const DefaultUserAgent =
 export const DefaultUserAgentAppendix = process.release?.name === 'node' ? `Node.js/${process.version}` : '';
 
 export const DefaultRestOptions = {
-	get agent() {
-		return new Agent({
-			connect: {
-				timeout: 30_000,
-			},
-		});
-	},
+	agent: null,
```

Comment:
I'm a bit concerned about this change getting forgotten in the future, while we're removing `undici` from `@discordjs/core`, are we also going to remove it from `discord.js`? If not, should we add a note (e.g. documenting this change) or a TODO so we don't forget to re-add this in the future?

Reply:
This change is a no-op for anyone not explicitly importing `DefaultRestOptions`.

https://github.com/discordjs/discord.js/blob/c6ca5a83e7b72613f95c2145606a1330f64ae894/packages/rest/src/lib/RequestManager.ts#L226-L231

It never gets used here. (though it is technically in RequestManager#options)
And furthermore, this default is useless unless someone sets timeout to a value greater than 30 seconds.

Reply:
Added to PR description as an example anyways so the changelog will show it